### PR TITLE
feat: set up NextJS 

### PR DESCRIPTION
**Pull request by amateima**
### Changes in this PR:

- set up NextJS framework
- add prettier besides the default ESLint configuration
⚠️ I followed the [instruction from here](https://nextjs.org/docs/basic-features/eslint#prettier), but `yarn lint` command is not picking up the rules I set in the [prettierrc file](https://github.com/UMAprotocol/website/blob/epic/new-website-boilerplate-setup/.prettierrc.json). Also VS Code doesn't show the warnings from Prettier
- added some [constants](https://github.com/UMAprotocol/website/blob/epic/new-website-boilerplate-setup/utils/constants.ts) that will be used
- added [static files](https://github.com/UMAprotocol/website/tree/epic/new-website-boilerplate-setup/public) which need to be served from the website address
⚠️ Let me know if I missed any files served in the old repo (PDFs, the Whitepaper doc, JSONs, etc)
- the only component I added is [GlobalStyles](https://github.com/UMAprotocol/website/tree/epic/new-website-boilerplate-setup/components/GlobalStyles) which configures the global css

❓I want to update the project settings in Vercel and change the `Framework Preset` option under `Build & Development Settings` section. I just want to make sure that this change is safe and it won't affect the current build that is deployed in production

Screenshot 2022-03-02 at 23 19 29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,101 +1,36 @@
-# UMA Press Kit Assets
+# UMA Website
 
-If you are looking for UMA `press-kit` assets, please see [`docs/press-kit`](https://github.com/UMAprotocol/website/tree/master/docs/press-kit).
```
Comment (daywiss) [inline README.md:3]: we shoudl probably not lose this copy

**Review by daywiss — COMMENTED** [review]
thanks for taking on this refactor! i know this is an incremental pr, but what should we expect on the preview link, currently its a broken page. if this is expected thats ok, just want to make sure.

**Review by daywiss — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (amateima) [reply README.md:3]: I added the files. Thx

File: components/GlobalStyles/typography.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { css } from "@emotion/react";
+
+export const typography = css`
+  @font-face {
+    font-family: "Halyard Display";
+    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2"),
+      url("/fonts/HalyardDisplay-Regular.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
```
Comment (Gamaranto) [inline components/GlobalStyles/typography.ts:10]: just a tip (nothing to change), you can save on bundle size by only importing latin characters, like we do in Across https://github.com/across-protocol/frontend-v1/blob/65abd7772704a9ec243fd370f9e8e76322f0905b/src/components/GlobalStyles/GlobalStyles.tsx#L14

**Review by Gamaranto — APPROVED** [review]
Looks good to me. 
Is the `vercel.json` file still in? Evaldo added it to fix some things about our headers, if it has been removed lets add it back in

**Comment by amateima** [comment]
> vercel.json

vercel json config file still in 👍

**Pull request merged**